src/index.less

```diff
@@ -0,0 +1,256 @@
+@chatbot-margin: 20px;
+@chatbot-border-radius: 30px;
+@chatbot-textarea-border-size: 1px;
+@chatbot-textarea-line-height: 18px;
+@chatbot-send-button-size: 24px;
+@chatbot-form-margin: 16px;
+
+@chatbot-header-height: 80px;
+@chatbot-body-height: 400px;
+
+@chatbot-footer-height:
+  @chatbot-form-margin * 2
+  + @chatbot-textarea-line-height * 3
+  + @chatbot-textarea-border-size * 2;
+
+@chatbot-width: 320px;
+@chatbot-height:  @chatbot-header-height + @chatbot-body-height + @chatbot-footer-height;
+@chatbot-floating-button-height: 54px;
+
+@chatbot-active-color: #01a3c0;
+@chatbot-disabled-color: #8f8f8f;
+
+@chatbot-bg-color: #fff;
+@chatbot-primary-color: #ed642a;
+@chatbot-header-bg-color1: #f98841;
+@chatbot-header-bg-color2: #f75a56;
+@chatbot-time-color: #bbbdbf;
+@chatbot-user-message-color: #fff;
+@chatbot-user-message-bg-color: #00a2bf;
+@chatbot-bot-message-bg-color: #f2f2f2;
+@chatbot-textarea-border-color: #babcbe;
+
+@chatbot-font-family: "Source Sans Pro";
+@chatbot-header-font-size: 24px;
+
+@chatbot-smallscreen: ~"only screen and (max-width: 767px) ";
+
+#ng-ovh-chatbot {
+  font-family: @chatbot-font-family;
+
+  button {
+    border: 0;
+    padding: 0;
+    cursor: pointer;
+    background: transparent;
+
+    svg {
+      width: 20px;
+      height: 20px;
+    }
+  }
+
+  .chatbot-floating-button {
+    position: fixed;
+    bottom: 0;
+    right: 0;
+    min-width: @chatbot-floating-button-height;
+    height: @chatbot-floating-button-height;
+    margin: @chatbot-margin;
+    z-index: 9998;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border-style: solid;
+    border-radius: 40px;
+    border-color: @chatbot-primary-color;
+    background: @chatbot-primary-color;
+
+    button {
```

Comment:
? This class already concerns a button

Reply:
Not exactly. It's a floating div wrapper around a button.

Reply:
Could you please change its name then ? :)